packages/flutter/test/material/user_accounts_drawer_header_test.dart

```diff
@@ -13,11 +13,13 @@ const Key avatarC = Key('C');
 const Key avatarD = Key('D');
 
 Future<void> pumpTestWidget(
-  WidgetTester tester, {
-  bool withName = true,
-  bool withEmail = true,
-  bool withOnDetailsPressedHandler = true,
-}) async {
+    WidgetTester tester, {
+      bool withName = true,
+      bool withEmail = true,
+      bool withOnDetailsPressedHandler = true,
+      Size otherAccountsPictureSize = const Size.square(40),
+      Size currentAccountPictureSize  = const Size.square(72),
```

Comment:
These two should have the `.0`, like `40.0` in order to be consistent with above.

Reply:
Or maybe they don't need defaults since UserAccountsDrawerHeader will set these defaults anyway?

Reply:
I do think we need them, that is if we want to be able to use the pumpTestWidget function for testing different size. If we don't specify them we get an error because of null-safety. I don't really know what is the best practice for this case...

Reply:
Sounds good, keeping it like this is ok but add the `.0`.

Reply:
Done :) ! @justinmc